### DISCUSSION TITLE
How to set the output folder

### DISCUSSION THREAD

**Discussion by jmcd040** [l1xnan/obsidian-better-export-pdf#198] [Q&A] [answered]
Somehow I set the output folder for PDFs to be my Print folder. But I've forgotten where to make that choice. I can't find a setting for it. Could someone remind me? Thank you.

**Comment by l1xnan** [comment, marked as answer]
It shouldn't be set up that way, it's just that the OS records the most recent folder.

**Reply by jmcd040** [reply]
Of course! I don't know why that slipped my mind. I guess I was so focused on finding a setting (Obsidian has so many!) rather than realizing the OS just remembered the last place I saved a file. Thanks!
